Composer/packages/tools/language-servers/language-generation/resources/memoryVariables.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
the format here is really confusing, can you use some sort of linting tool on json?